fhir-examples/src/main/resources/ibm-json.txt

```diff
@@ -9,7 +9,6 @@ OK         json/ibm/complete-absent/AdverseEvent-1.json
 OK         json/ibm/complete-absent/AllergyIntolerance-1.json
 OK         json/ibm/complete-absent/AllergyIntolerance-2.json
 OK         json/ibm/complete-absent/AllergyIntolerance-3.json
-OK         json/ibm/complete-absent/AllergyIntolerance-4.json
 OK         json/ibm/complete-absent/AllergyIntolerance-5.json
```

Comment:
Is there a way to avoid the gap in numbering?

Reply:
There might be... but it means 4 is a dupe of 1,2,3